src/pages/[locale]/buildings/index.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <section class="default-layout__info building-overview">
+    <h2 class="a11y-sr-only">
+      {{ title }}
+    </h2>
+
+    <BuildingCard
+      v-for="building in buildings"
+      :key="building.slug"
+      :building="building"
+    />
+  </section>
+</template>
+
+<script setup lang="ts">
+import { storeToRefs } from "pinia";
+import { useI18n } from "vue-i18n";
+import metaHead from "~/lib/meta-head";
+import { useStore } from "~/stores/store";
+import { useMapStore } from "~/stores/map";
+
+definePageMeta({
+  alias: "/:locale/gebouwen",
+});
+
+const store = useStore();
+const mapStore = useMapStore();
+const { buildings } = storeToRefs(store);
+const { t } = useI18n();
+
+const title = computed(() => t("buildingTitle"));
+
+useHead(() =>
+  metaHead({
+    title: title.value,
+    description: "",
+  })
+);
+
+onMounted(() => {
+  store.clearSelection();
+  mapStore.zoomToCampus();
+  mapStore.getMap().then(() => mapStore.updateMarkers());
```

Comment:
since both `.getMap()` and `.updateMarkers()` reside in `mapStore` maybe make the `.getMap()` part of the `.updateMarkers()` method so the code here could be simplified to:

```ts
onMounted(() => {
  store.clearSelection();
  mapStore.zoomToCampus();
  mapStore.updateMarkers();
```